packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -134,7 +134,7 @@ module.exports = {
           /\.json$/,
           /\.svg$/
         ],
-        loader: 'url',
+        loader: 'url-loader',
         query: {
```

Comment:
Although webpack 2 allows `query`, it has chosen to use `options` in its docs which IMO is better, would you consider changing?